src/anm/analytics.js

```diff
@@ -33,40 +43,41 @@ var Analytics = function () {
     this.trackPlayingComplete = this.trackPlayer('playing_complete');
 };
 
-Analytics.prototype.track = function track(name, opts) {
-    opts = opts || {};
-    opts.name = name;
-    opts.referer = document.referrer;
-    opts.lang = navigator.language || navigator.userLanguage;
-    opts.url = location.href;
-    opts.screenHeight = screen.height;
-    opts.screenWidth = screen.width;
-    opts.windowHeight = window.innerHeight;
-    opts.windowWidth = window.innerWidth;
-    opts.timestamp = new Date().getTime();
+Analytics.prototype.track = function track(name, player, action) {
     if (this.enabled) {
-        this.queue.push(opts);
+        this.queue[player.viewId] = this.queue[player.viewId] || {
```

Comment:
It would be nicer to create some separate method for collecting this data (id's and screen metrics and everything else) first time for every player.